app/models/financial_transaction_class.rb

```diff
@@ -1,5 +1,6 @@
 class FinancialTransactionClass < ApplicationRecord
   has_many :financial_transaction_types, dependent: :destroy
+  has_many :suppliers
```

Comment:
Does it make sense to add `dependent` here?

Reply:
added